modules/sms/src/providers/messageBird.ts

```diff
@@ -0,0 +1,65 @@
+import { ISmsProvider } from '../interfaces/ISmsProvider';
+import ConduitGrpcSdk from '@conduitplatform/grpc-sdk';
+import messagebird from 'messagebird';
+
+export class messageBirdProvider implements ISmsProvider {
+  private readonly accessKeyId: string;
+  private client: messagebird.MessageBird;
+  private originatorName: string;
+  constructor(settings: { accessKeyId: string; originatorName: string }) {
+    ({ accessKeyId: this.accessKeyId, originatorName: this.originatorName } = settings);
+    this.client = require('messagebird').initClient(this.accessKeyId);
+  }
+
+  async sendSms(phoneNumber: string, message: string) {
+    await this.client.messages.create(
```

Comment:
you need to wrap this in a promise as you do in the other 2 functions of messagebird since they are not async